python/python/glide/async_commands/core.py

```diff
@@ -362,6 +362,11 @@ class InsertPosition(Enum):
     AFTER = "AFTER"
 
 
+class FlushMode(Enum):
+    ASYNC = "ASYNC"
+    SYNC = "SYNC"
```

Comment:
add since to SYNC mode (I'll do this for java)

Reply:
not sure if i did it right, should the since be added to the function doc or the enum's doc?